app/views/contexts/section_summary_context.py

```diff
@@ -11,10 +12,21 @@
 class SectionSummaryContext(Context):
     def __call__(self, current_location):
         summary = self._build_summary(current_location)
+        title_for_location = self._title_for_location(current_location)
+        title = (
+            self._placeholder_renderer.render_placeholder(
+                title_for_location, current_location.list_item_id
+            )
+            if isinstance(title_for_location, dict)
+            else title_for_location
+        )
 
         return {
             "summary": {
-                "title": self._title_for_location(current_location),
+                "title": title,
+                "page_title": safe_content(
+                    f'{self._schema.get_single_string_value(title_for_location)} - {self._schema.json["title"]}'
+                ),
```

Comment:
Can we keep this as a method call to `_get_safe_page_title` as I think it will improve readability and make it consistent with other places in runner?